src/main/java/com/iridium/iridiumskyblock/commands/DeleteCommand.java

```diff
@@ -54,6 +67,17 @@ public boolean execute(CommandSender sender, String[] args) {
         return false;
     }
 
+    private void deleteOtherIsland(CommandSender commandSender, String playerName) {
```

Comment:
not sure if I like this being a seperate method

Reply:
Sure, I can inline it, but imo it's much more readable this way

Reply:
It would look like this: https://pasty.lus.pm/UuHIyP